components/button/test/index.js

```diff
@@ -103,17 +104,12 @@ describe( 'Button', () => {
 		} );
 	} );
 
-	// Disable reason: This test is desirable, but unsupported by Enzyme in
-	// the current version, as it depends on features new to React in 16.3.0.
-	//
-	// eslint-disable-next-line jest/no-disabled-tests
-	describe.skip( 'ref forwarding', () => {
+	describe( 'ref forwarding', () => {
 		it( 'should enable access to DOM element', () => {
 			const ref = createRef();
 
-			mount( <ButtonWithForwardedRef ref={ ref } /> );
-
-			expect( ref.current.nodeName ).toBe( 'button' );
+			TestUtils.renderIntoDocument( <ButtonWithForwardedRef ref={ ref } /> );
```

Comment:
It would read a bit nicer If we use destructuring when importing this method to avoid TestUtils at all in this file.

Reply:
TestUtils is the default export from `react-dom/test-utils`.  So I'd have to destructure on a subsequent statement.  If `renderIntoDocument` was available as a direct import I'd definitely do it but its not.  For now, I think it's a bit useful knowing at a glance that `renderIntoDocument` is part of TestUtils?

Reply:
I wasn't aware of it, it makes sense 👍